test/test_proxy_tensor.py

```diff
@@ -1498,6 +1498,7 @@ def f(lengths, values):
             # tolist not directly supported atm
             sizes = [lengths[i].item() for i in range(lengths.size(0))]
             for s in sizes:
+                # TODO(avik): no assertion generated with torch._check_is_size?
```

Comment:
Is this resulting in an error? It seems like this is because `torch._check_is_size` is a softer version of `_constrain_as_size`, where it will only add assertions if we cannot prove something at compile-time. https://fburl.com/code/sm5ojgw2

Reply:
Oh interesting, deserves a follow-up.